src/interp/interp.cc

```diff
@@ -386,6 +401,11 @@ Result DefinedFunc::DoCall(Thread& thread,
   result = thread.Run(out_trap);
   if (result == RunResult::Trap) {
     return Result::Error;
+  } else if (result == RunResult::Exception) {
```

Comment:
This is possible, but it's not trivial to add here because we need to track the currently thrown exception (only caught exceptions are stored currently, for rethrow) and then access it here with a new helper function to print the values to a string (`WriteValues` exists, but only operates on streams for tool/test output).

Maybe this could be done in follow-up changes later? WDYT?

Reply:
Yeah that sounds fine. (Also I think doing that is not a high priority)